README.md

```diff
@@ -10,12 +10,47 @@ python stix_shifter.py `<data source>` `<input format>` `<STIX pattern>`
 
 Example of converting a STIX pattern to an AQL query:
 
-`python3 stix_shifter.py "qradar" "sco" "[ipv4-addr:value = '192.168.122.83' or ipv4-addr:value = '192.168.122.84']"`
+`python stix_shifter.py "qradar" "sco" "[ipv4-addr:value = '192.168.122.83' or ipv4-addr:value = '192.168.122.84']"`
```

Comment:
Python 3 is still required, but I would think most people would have the python command pointing to python 3 and so that's why I suggested it be removed from the readme. Maybe I'm wrong on that.